src/test/kotlin/ar/edu/unahur/obj2/servidorWeb/ServidorWebTest.kt

```diff
@@ -2,9 +2,98 @@ package ar.edu.unahur.obj2.servidorWeb
 
 import io.kotest.core.spec.style.DescribeSpec
 import io.kotest.matchers.shouldBe
+import java.net.URL
 import java.time.LocalDateTime
 
 class ServidorWebTest : DescribeSpec({
   describe("Un servidor web") {
+    val servidorWeb = ServidorWeb()
+    val extensionesImagen = setOf("jpg", "png")
+    val extensionesDoc = setOf("docx", "txt")
+    val ipsSospechosas = mutableSetOf("102.46.81.0", "102.20.20.0")
+    val moduloImagen = Modulo(extensionesImagen, "Imagen", 15 )
+    val moduloDocumento = Modulo(extensionesDoc, "Documento", 17)
+    val analizadorDemora = AnalizadorDeDemoraEnRespuesta(12)
+    val analizadorIp = AnalizadorDeIPsSospechosas(ipsSospechosas)
+    val analizadorEst = AnalizadorDeEstadisticas()
+    val pedido01 = Pedido("103.57.91.1", URL("http://pepito.com.ar/hola.txt"), LocalDateTime.now())
+    val pedido02 = Pedido("102.46.81.0", URL("https://pepito.com.ar/hola.docx"), LocalDateTime.now())
+    val pedido03 = Pedido("102.20.20.0", URL("http://pepito.com.ar/hola.wmv"), LocalDateTime.now())
+    val pedido04 = Pedido("102.46.81.0", URL("http://pepito.com.ar/hola.jpg"), LocalDateTime.now())
+    val pedido05 = Pedido("102.20.20.0", URL("http://pepito.com.ar/hola.png"),
+      LocalDateTime.of(2021, 6, 19, 10, 11, 0))
+    val pedido06 = Pedido("102.20.20.0", URL("http://pepito.com.ar/hola.jpg"),
+      LocalDateTime.of(2021, 6, 18, 10, 11, 0))
+
+    servidorWeb.agregarModulo(moduloImagen)
+    servidorWeb.agregarModulo(moduloDocumento)
+    servidorWeb.agregarAnalizador(analizadorDemora)
+    servidorWeb.agregarAnalizador(analizadorIp)
+    servidorWeb.agregarAnalizador(analizadorEst)
+
+    describe("Una respuesta con código 200") {
+      it("recibirPedido") {
+        val respuesta = servidorWeb.recibirPedido(pedido01)
+        respuesta.codigo.shouldBe(CodigoHttp.OK)
+      }
+    }
+
+    describe("Una respuesta con código 501") {
+      it("recibirPedido") {
+        val respuesta = servidorWeb.recibirPedido(pedido02)
+        respuesta.codigo.shouldBe(CodigoHttp.NOT_IMPLEMENTED)
+      }
+    }
+
+    describe("Una respuesta con código 404") {
+      it("recibirPedido") {
+        val respuesta = servidorWeb.recibirPedido(pedido03)
+        respuesta.codigo.shouldBe(CodigoHttp.NOT_FOUND)
+      }
+    }
```

Comment:
Estos 3 yo los pondría en el mismo describe "recibirPedido" y después un it por respuesta, para denotar que prueban el mismo método